test/repositories/SessionRepositorySpec.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package repositories
+
+import base.BaseSpec
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class SessionRepositorySpec extends BaseSpec {
```

Comment:
There would appear to be an absence of tests. For external-guidance Mac hasn't created any tests for the mongo components, but he did create some tests for the formatter. However, as you are using a formatter provided by Mongo perhaps such test are unnecessary.

If you don't think any tests are required at present I'd just add a comment to that effect.

Reply:
I don't think unit tests are required for the repositories as long as integration tests exist. Unit testing the repository (as it inherits almost all the functionality from it's super class) is redundant.

Reply:
Yes, I left this empty Spec file, but added $COVERAGE-OFF to the code module